src/frontend/src/components/render/ModelType.tsx

```diff
@@ -0,0 +1,130 @@
+import { t } from '@lingui/macro';
+
+export enum ModelType {
+  part = 'part',
+  supplierpart = 'supplierpart',
+  manufacturerpart = 'manufacturerpart',
+  partcategory = 'partcategory',
+  stockitem = 'stockitem',
+  stocklocation = 'stocklocation',
+  build = 'build',
+  company = 'company',
+  purchaseorder = 'purchaseorder',
+  salesorder = 'salesorder',
+  salesordershipment = 'salesordershipment',
+  returnorder = 'returnorder',
+  address = 'address',
+  contact = 'contact',
+  owner = 'owner',
+  user = 'user'
+}
+
+interface ModelInformatonInterface {
+  label: string;
+  label_multiple: string;
+  url_overview: string;
+  url_detail: string;
+}
+
+type ModelDictory = {
+  [key in keyof typeof ModelType]: ModelInformatonInterface;
+};
+
+export const ModelLinks: ModelDictory = {
+  part: {
+    label: t`Part`,
+    label_multiple: t`Parts`,
```

Comment:
not sure how that would be applicable here, we also split the translation in the backend models.

Reply:
This was more of a note, as I saw we already do this everywhere, but the i18n engine offers a better way for [pluralization](https://lingui.dev/guides/plurals). So we could consider using that. I think this will also be shown better in crowdin, so the translator can translate all forms at once and not the singular yet and the plural a few minutes later. Also for some languages there are other rules for pluralization of which crowdin and lingui would take care better than we do.